src/sl-vue-tree.js

```diff
@@ -25,6 +25,13 @@ export default {
       type: Boolean,
       default: true
     },
+    multiSelectKey: {
+      type: String,
+      default: 'ctrlKey',
```

Comment:
I think the default value should be something like ['ctrlKey', 'metaKey'] to be consistent for Mac OS

Reply:
But is it possible to provide an array of keys for a String prop ? I don't think so. I chose 'ctlrKey' because it is what it used now (hence the PR doesn't break anything). But it opens up the possibility to customize it.

Do you want me to change something on the PR ?

Reply:
I mean, it looks like the multiple selection with `cmd` key is not working for MacOS users by default for now. And your PR allows to fix that. And that have to be fixed by default without any additional settings.

Some idea how to do that:
```js
multiSelectKey: {
 type: [String, Array],
 default: ['ctrlKey', 'metaKey'],
 validator: function (value) {
    let allowedKeys = ['ctrlKey', 'metaKey', 'altKey'];
    let multiselectKeys = Array.isArray(value) ? value : [multiselectKey];
    multiselectKeys = multiselectKeys.filter(keyName => allowedKeys.indexOf(keyName ) !== -1 );
    return !!multiselectKeys.length;
 }
}

.....

select(path, addToSelection = false, event = null) {
  const multiselectKeys = Array.isArray(this.multiSelectKey) ?
     this.multiSelectKey:
     [this.multiSelectKey];
  const multiselectKeyIsPressed = event && !!multiselectKeys.find(key => event[key]);
  const addToSelection = (multiselectKeyIsPressed || addToSelection) && this.allowMultiselect ;
 ........

````

Reply:
I got it ! Sorry, I was stuck inside my "keep the thing working as they are for now". But of course it's much better to have defaults working everywhere. And yes, you can provide multiple prop types, thanks. :-) I am updating the PR code with yours.

Reply:
Code pushed. Note that I changed also the case of the prop name to match that of 'allowMultiselect'.